### ISSUE TITLE
New submissions for Fri, 30 Jun 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#49]
# New submissions for Fri, 30 Jun 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送
See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### As Simple as Possible but No Simpler: Optimizing the Performance of  Neural Net Emulators for Galaxy SED Fitting
 - **Authors:** Elijah P. Mathews, Joel Leja, Joshua S. Speagle, Benjamin D. Johnson, Justus Gibson, Erica J. Nelson, Katherine A. Suess, Sandro Tacchella, Katherine E. Whitaker, Bingjie Wang
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2306.16442
 - **Pdf link:** https://arxiv.org/pdf/2306.16442
 - **Abstract**
 Artificial neural network emulators have been demonstrated to be a very computationally efficient method to rapidly generate galaxy spectral energy distributions (SEDs), for parameter inference or otherwise. Using a highly flexible and fast mathematical structure, they can learn the nontrivial relationship between input galaxy parameters and output observables. However, they do so imperfectly, and small errors in flux prediction can yield large differences in recovered parameters. In this work, we investigate the relationship between an emulator's execution time, uncertainties, correlated errors, and ability to recover accurate posteriors. We show that emulators can recover consistent results to traditional fits, with precision of $25\!-\!40\%$ in posterior medians for stellar mass, stellar metallicity, star formation rate, and stellar age. We find that emulation uncertainties scale with an emulator's width $N$ as $\propto N^{-1}$ while execution time scales as $\propto N^2$, resulting in an inherent tradeoff between execution time and emulation uncertainties. We also find that emulators with uncertainties smaller than observational uncertaities are able to recover accurate posteriors for most parameters without a significant increase in catastrophic outliers. Furthermore, we demonstrate that small architectures can produce flux residuals that have significant correlations, which can create dangerous systematic errors in colors. Finally, we show that the distributions chosen for generating training sets can have a large effect on emulators' ability to accurately fit rare objects. Selecting the optimal architecture and training set for an emulator will minimize the computational requirements for fitting near-future large-scale galaxy surveys.
#### Light and colour of cirrus, translucent and opaque dust in the  high-latitude area of LDN 1642
 - **Authors:** K. Mattila (1), P. Väisänen (2 and 3), K. Lehtinen (1), L. Haikala (4), M. Haas (5) ((1) Department of Physics, University of Helsinki, (2) South African Astronomical Observatory, Cape Town, (3) Southern African Large Telescope, Cape Town, (4) Instituto de Astronomía y Ciencias Planetarias, Universidad de Atacama, Copiapo, (5) Astronomisches Institut, Ruhr-Universität Bochum)
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2306.16467
 - **Pdf link:** https://arxiv.org/pdf/2306.16467
 - **Abstract**
 We have performed a 5-colour surface photometric study of the high-galactic-latitude area of dark nebula LDN 1642. Scattered light properties are presented of diffuse, translucent and opaque dust over the range of 3500 -- 5500 A. Far infrared absolute photometry at 200 um improves the precision of and provides a zero point to the extinction. The intensity of the scattered light depends on dust column density in a characteristic way: for optically thin dust the intensity first increases linearly, then turns to a saturation value; at still larger extinctions the intensity turns down to a slow decrease. The $A_V$ value of the saturated intensity maximum shifts in a systematic way, from $A_V\approx$ 1.5 mag at 3500 A, to $\sim 3$ mag at 5500 A. The intensity curves offer a straight-forward explanation for the behaviour of the scattered-light colours. At the intensity peak the colour agrees with the integrated starlight colour, while it is bluer at the low- and redder at the high-column-density side of the peak, respectively. These colour changes are a direct consequence of the wavelength dependence of the extinction. We have compared the colours of the LDN 1642 area with other relevant observational studies: high-latitude diffuse/translucent clouds, wide-field cirrus dust; and externally illuminated AGB-star envelopes. For extragalactic low-surface-brightness sources cirrus is an unwanted foreground contaminant. Our results for cirrus colours can help to distinguish cases where a diffuse plume or stream, apparently associated with a galaxy or a group or cluster, is more likely a local cirrus structure. Keywords: ISM: dust, extinction -- ISM: clouds, individual LDN 1642 -- Galaxy: solar neighbourhood -- Astronomical instruments, methods and techniques: methods -- Physical data and processes: scattering
#### Spectroscopic comparative study of the red giant binary system gamma  Leonis A and B
 - **Authors:** Yoichi Takeda
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); Earth and Planetary Astrophysics (astro-ph.EP)
 - **Arxiv link:** https://arxiv.org/abs/2306.16723
 - **Pdf link:** https://arxiv.org/pdf/2306.16723
 - **Abstract**
 Gamma~Leo is a long-period visual binary system consisting of K0III (A) and G7III (B) giants, in which particular interest is attracted by the brighter A since the discovery of a planet around it. While detailed spectroscopic comparative study of both components would be worthwhile (e.g., for probing any impact of planet formation on chemical abundances), such a research seems to have been barely attempted as most available studies tend to be biased toward A. Given this situation, the physical properties of A and B along with their differences were investigated based on high-dispersion spectra in order to establish their stellar parameters, evolutionary status, and surface chemical compositions. The following results were obtained. (1) The masses were derived as ~1.7Msun and ~1.6Msun for A and B, respectively, both of which are likely to be in the stage of red clump giants after He-ignition. The mass of the planet around A has also been revised as m*sin(i) = 10.7M_Jupiter (increased by ~20%). (2) These are normal giants of subsolar metallicity ([Fe/H]~-0.4) belonging to the thin-disk population. (3) A as well as B show moderate C deficiency and N enrichment, which are in compatible with the prediction from the standard stellar evolution theory. (4) The chemical abundances of 26 elements are practically the same within <~0.1dex for both components, which implies that the surface chemistry is not appreciably affected by the existence of a planet in A.
#### A Dynamically Distinct Stellar Population in the Leading Arm of the  Sagittarius Stream
 - **Authors:** Gwibong Kang, Young Sun Lee, Young Kwang Kim, Timothy C. Beers
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2306.16748
 - **Pdf link:** https://arxiv.org/pdf/2306.16748
 - **Abstract**
 We present a chemical and dynamical analysis of the leading arm (LA) and trailing arm (TA) of the Sagittarius (Sgr) stream, as well as for the Sgr dwarf galaxy core (SC), using red giant branch, main sequence, and RR Lyrae stars from large spectroscopic survey data. The different chemical properties among the LA, TA, and SC generally agree with recent studies, and can be understood by radial metallicity gradient established in the progenitor of the Sgr dwarf, followed by preferential stellar stripping from the outer part of the Sgr progenitor. One striking finding is a relatively larger fraction of low-eccentricity stars (e < 0.4) in the LA than in the TA and SC. The TA and SC exhibit very similar distributions. Considering that a tidal tail stripped off from a dwarf galaxy maintains the orbital properties of its progenitor, we expect that the e-distribution of the LA should be similar to that of the TA and SC. Thus, the disparate behavior of the e-distribution of the LA is of particular interest. Following the analysis of Vasiliev et al., we attempt to explain the different e-distribution by introducing a time-dependent perturbation of the Milky Way by the Large Magellanic Cloud (LMC)'s gravitational pull, resulting in substantial evolution of the angular momentum of the LA stars to produce the low-e stars. In addition, we confirm from RR Lyrae stars with high eccentricity (e > 0.6) that the TA stars farther away from the SC are also affected by disturbances from the LMC.
#### Neutrino flares of radio blazars observed from TeV to PeV
 - **Authors:** Alisa Suray, Sergey Troitsky
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2306.16797
 - **Pdf link:** https://arxiv.org/pdf/2306.16797
 - **Abstract**
 Radio blazars have been linked both to individual high-energy neutrino events and to excesses in likelihood sky maps constructed from lower-energy neutrino data. However, the exact mechanism by which neutrinos are produced in these sources is still unknown. Here, we demonstrate that IceCube neutrinos with energies over 200 TeV, which were previously associated with bright radio blazars, are significantly more likely to be accompanied by flares of lower-energy events, compared to those lacking blazar counterparts. The parsec-scale core radio flux of blazars positioned within the error regions of energetic events is strongly correlated with the likelihood of a coincident day-scale lower-energy neutrino flare reported by IceCube. The probability of a chance correlation is 3.6*10^{-4}. This confirms the neutrino-blazar connection in a new and independent way, and provides valuable clues to understanding the origin of astrophysical neutrinos.
#### Gravitational collapse and accretion flows in the hub filament system  G323.46-0.08
 - **Authors:** Yingxiu Ma, Jianjun Zhou, Jarken Esimbek, Willem Baan, Dalei Li, Xindi Tang, Yuxin He, Weiguang Ji, Dongdong Zhou, Gang Wu, Kadirya Tursun, Toktarkhan Komesh
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2306.16875
 - **Pdf link:** https://arxiv.org/pdf/2306.16875
 - **Abstract**
 We studied the hub filament system G323.46-0.08 based on archival molecular line data from the SEDIGISM 13CO survey and infrared data from the GLIMPSE, MIPS, and Hi-GAL surveys. G323.46-0.08 consists of three filaments, F-north, F-west, and F-south, that converge toward the central high_mass clump AGAL 323.459-0.079. F-west and Part 1 of the F-south show clear large-scale velocity gradients 0.28 and 0.44 km s-1 pc-1, respectively. They seem to be channeling materials into AGAL 323.459-0.079. The minimum accretion rate was estimated to be 1216 M Myr-1. A characteristic V-shape appears around AGAL 323.459-0.079 in the PV diagram, which traces the accelerated gas motions under gravitational collapse. This has also been supported by model fitting results. All three filaments are supercritical and they have fragmented into many dense clumps. The seesaw patterns near most dense clumps in the PV diagram suggests that mass accretion also occurs along the filament toward the clumps. Our results show that filamentary accretion flows appear to be an important mechanism for supplying the materials necessary to form the central high-mass clump AGAL 323.459-0.079 and to propel the star forming activity taking place therein.
#### Cross section for supernova axion observation in neutrino water  Cherenkov detectors
 - **Authors:** Pierluca Carenza, Giampaolo Co', Maurizio Giannotti, Alessandro Lella, Giuseppe Lucente, Alessandro Mirizzi, Thomas Rauscher
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE); High Energy Physics - Phenomenology (hep-ph); Nuclear Theory (nucl-th)
 - **Arxiv link:** https://arxiv.org/abs/2306.17055
 - **Pdf link:** https://arxiv.org/pdf/2306.17055
 - **Abstract**
 Axions coupled to nucleons might be copiously emitted from core-collapse supernovae (SNe). If the axion-nucleon coupling is strong enough, axions would be emitted from the SN as a burst and, reaching Earth, may excite the oxygen nuclei in water Cherenkov detectors (${}^{16}{\rm O} + a \to {}^{16}{\rm O}^{*}$). This process will be followed by radiative decay(s) of the excited state (${}^{16}{\rm O}^* \rightarrow {}^{16}{\rm O}+\gamma $) providing a strategy for a direct detection of axions from a Galactic SN in large underground neutrino Cherenkov detectors. Motivated by this possibility, we present an updated calculation of axion-oxygen cross section obtained by using self-consistent continuum Random Phase Approximation. We calculate the branching ratio of the oxygen nucleus de-excitation into gamma-rays, neutrons, protons and $\alpha$-particles. These results are used to revisit the detectability of axions from SN 1987A in Kamiokande-II.
#### A rare outburst from the stealthy BeXRB system Swift J0549.7-6812
 - **Authors:** M.J. Coe (Southampton), J.A. Kennea (PSU), I.M. Monageng (UCT), D.A.H. Buckley (SAAO), A. Udalski (Warsaw), P.A. Evans (Leicester)
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2306.17098
 - **Pdf link:** https://arxiv.org/pdf/2306.17098
 - **Abstract**
 Swift J0549.7-6812 is an Be/X-ray binary system (BeXRB) in the Large Magellanic Cloud (LMC) exhibiting a 6s pulse period. Like many such systems the variable X-ray emission is believed to be driven by the underlying behaviour of the mass donor Be star. In this paper we report on X-ray observations of the brightest known outburst from this system which reached a luminosity of 8 x 10^37 erg/s. These observations are supported by contemporaneous optical photometric observations, the first reported optical spectrum, as well as several years of historical data from OGLE and GAIA. The latter strongly suggest a binary period of 46.1d. All the observational data indicate that Swift J0549.7-6812 is a system that spends the vast majority of its time in X-ray quiescence, or even switched off completely. This suggests that occasional observations may easily miss it, and many similar systems, and thereby underestimate the massive star evolution numbers for the LMC.

by olozhika (Xing Yuchen). 

2023-06-30
